tutorials/haskell/hspec-golden-test-tutorial/code/test/Hello/HelloGoldenSpec.hs

```diff
@@ -0,0 +1,12 @@
+module Hello.HelloGoldenSpec where
+
+import           Test.Hspec                -- Test
+import           Test.Hspec.Golden         -- Golden Tests
+import           HelloWorld                -- SUT
+
+
+spec :: Spec
+spec =
+    describe "sayHi" $
+    it "returns Hello Golden Testers string" $
+    defaultGolden "hello" sayHi
```

Comment:
@Jagl257 The test looks fine. However, let's indent it, sounds good?
```hs
    describe "sayHi" $
      it "returns Hello Golden Testers string" $
        defaultGolden "hello" sayHi
```

Reply:
Sounds great, done.